cfgov/jinja2/v1/_includes/organisms/full-width-text.html

```diff
@@ -0,0 +1,22 @@
+
+{# ==========================================================================
+
+   full_width_text.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create a gray well organism.
```

Comment:
Oops, doesn't look like it got fixed in https://github.com/cfpb/cfgov-refresh/commit/71d1b9cf55cf8a6c8b57b282e0ad76b7876e7252 It's minor though if this needs to go through. Open an issue though if it's not fixed.